REST/RESTListener+Replicate.cc

```diff
@@ -289,17 +311,19 @@ namespace litecore { namespace REST {
                          canceled ? "Stopped" : "No matching task");
             return;
         }
-
+        // Auth:
+        slice authType = params["authtype"].asString();
+        if (authType) {
+            slice user = params["user"].asString();
+            slice psw = params["password"].asString();
+            task->setAuth(authType, user, psw);
+        }
+        task->setStopOnIdle(params["stopOnIdle"].asBool());
         task->start(localDB, localName,
                     remoteAddress, remoteDbName,
                     pushMode, pullMode);
 
         HTTPStatus statusCode = HTTPStatus::OK;
-        if (!continuous) {
```

Comment:
One-shot replication requests need to be synchronous, because that's the way the REST API is defined. This code is necessary for that. Don't remove it.